features/nanostack/mbed-mesh-api/source/WisunInterface.cpp

```diff
@@ -119,13 +131,18 @@ nsapi_error_t WisunInterface::configure()
     }
 #endif
 
-#ifdef MBED_CONF_MBED_MESH_API_WISUN_NETWORK_SIZE
-    status = set_network_size(MBED_CONF_MBED_MESH_API_WISUN_NETWORK_SIZE);
-    if (status != MESH_ERROR_NONE) {
-        tr_error("Failed to set network size");
+    ret_val = mesh_kcm_wisun_network_size_init(&network_size);
```

Comment:
Previously it was Ok not to have configuration as it would use stack defaults you haven changed that, but I think it should be working like previously

Reply:
Ok, I'll change that back to original.
So if there is a configuration error in KCM data, we use Mbed OS defaults. If Mbed OS default does not exists, we use Nanostack defaults.